### PR TITLE
Minimal frontpage

### PR DESCRIPTION
**Pull request by thijskh**
This is a first attempt at implementing the proposed solution to #1515: replace the public front page with a config setting to allow the admin to redirect people somewhere else, otherwise display a very basic message and that's it.

Screenshot of the new minimal front page

I did consider to add a "login as admin" on the top-right but decided against it for now, as per @jaimeperez' comments in 1515.

It's not per se the definitive form, I hope some concrete implementation helps us getting along to decide if this is the direction that we want and if so, how exactly.

### CODE REVIEW

File: config-templates/config.php [hunk 1]
```diff
@@ -853,6 +851,12 @@
         ],
     ],
 
+    /**
+     * Set to a full URL if you want to rediret users that land on SimpleSAMLphp's
```
Comment (tvdijen) [inline config-templates/config.php:855]: typo

**Comment by tvdijen** [comment]
This is a no-brainer merge for me, as it takes away the vulnerability of exposing individual authsources.
Perhaps we should consider an 'OpenConext'-like landing-page where we add links to certs + metadata for all entities.

**Review by tvdijen — APPROVED** [review]

**Comment by thijskh** [comment]
This removes all code whereby anyone from the general public can (try to) authenticate to any auth source.

You will still keep the option to test auth sources in the admin module of course.

**Comment by tvdijen** [comment]
One more thing that caught my eye is that your header is in Dutch and the rest of the text is in English (screenshot).

**Comment by thijskh** [comment]
That is because I have not yet added translations for the new strings.

**Comment by tvdijen** [comment]
> This removes all code whereby anyone from the general public can (try to) authenticate to any auth source.
This is very very good!

> You will still keep the option to test auth sources in the admin module of course.
Yes, but you can't let a 3rd party user authenticate to get an idea on what attributes they're releasing.. One could argue that this can be done using debug-logging, but that's not something you generally use on a production environment..

I'm thinking more like an IDP-first scenario where you lack to specify the SP-entityid and instead of a redirect to the SP you get an overview of the attributes.. This would mean instead of displaying local authsources, you'd display local/remote IDPs.. I guess what I'm after is not that different from the OpenConext debug-page.

**Comment by thijskh** [comment]
We could make it a module that you can enable?

**Comment by tvdijen** [comment]
Yes, that'd be wonderful!

**Comment by tvdijen** [comment]
It's gonna take some time cleaning up the translations for the core-module.
Some translations now belong to the admin-module (these changes can go straight to `master`),
others have become ununsed and can be removed as part of this PR.

My approach so far has been:
- Search for the fully translated English string to see if it's being used anywhere outside the .po-files; if yes, see if it needs to be moved, if no remove from all .po-files.
- Search for the 'old-style'-tag and use the same approach as above.
- Note that some translations only exist as tag in the .po-files. Try not to remove good translations in the proces.

**Comment by thijskh** [comment]
Well, no one has added any translations to the admin module until now it seems...

**Comment by tvdijen** [comment]
> Well, no one has added any translations to the admin module until now it seems...

I've made a start in `master`.. I found one tag that existed in the base translations and moved it to `admin`... It's a horrible job.

**Comment by thijskh** [comment]
I've automatically migrated all strings that had a full string equivalent in the old files for all languages.

Only thing that is not automatically migrated is when the old files have a "namespaced" translation tag but not the full string version. If we want to port this, we need a mapping of these old tags to the new strings.

**Comment by tvdijen** [comment]
Just so you know, I try and process a couple of tags every day, but I still have two weeks of work left if I continue at this pace...

**Comment by tvdijen** [comment]
Finally done!  I suggest we merge this (please DO NOT squash as the commit would be HUGE).
Let's merge and tag `beta.6`.. Let's keep the issue open as a reminder we have to make a module to replace old behaviour..

**Pull request merged**